ci/install_musl.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright (c) 2020 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+install_aarch64_musl() {
+	local arch=$(uname -m)
+	if [ "${arch}" == "aarch64" ]; then
+		local musl_tar="${arch}-linux-musl-native.tgz"
+		local musl_dir="${arch}-linux-musl-native"
+		poshd /tmp
+		curl -sLO https://musl.cc/${musl_tar}
```

Comment:
Nit: Running network operation as `root`? Or is it: I thought we used `sudo` in all these scripts to become `root`?

Reply:
Good point! Thanks!

Reply:
Turns out we don't have sudo there. Let me retry and see if we are already running root in CI.